clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -307,11 +307,9 @@ Changes in existing checks
 - Improved :doc:`misc-const-correctness
   <clang-tidy/checks/misc/const-correctness>` check to avoid false positive when
   using pointer to member function.
-
-- Improved :doc:`misc-const-correctness
-  <clang-tidy/checks/misc/const-correctness>` check to not warn on uses in
-  type-dependent binary operators, when the variable that is being
-  looked at, is not the dependent operand.
+  Additionally, the check no longer emits a diagnostic when
```

Comment:
Looks fine, but wrap this after "to member function."

Reply:
Like this?